lib/vulcan.js

```diff
@@ -222,7 +222,8 @@ Vulcan.prototype = {
   isInsideTemplate: dom5.predicates.parentMatches(
       dom5.predicates.hasTagName('template')),
 
-  flatten: function flatten(tree, isMainDoc) {
+  flatten: function flatten(tree, rootDocPath) {
+    var isMainDoc = tree.href === rootDocPath;
```

Comment:
Are both `tree.href` and `rootDocPath` normalized at this point such that equality is guaranteed for equivalent paths?